csrfguard/src/main/java/org/owasp/csrfguard/CsrfGuard.java

```diff
@@ -403,6 +409,10 @@ private ConfigurationProvider retrieveNewConfig() {
         final ConfigurationProviderFactory configurationProviderFactory = CsrfGuardUtils.newInstance(configurationProviderFactoryClass);
 
         configurationProvider = configurationProviderFactory.retrieveConfiguration(this.properties);
+        if (initialiseJavaScriptConfigurationForNewConfigurationInstances) {
```

Comment:
please prefix class fields with `this` for readability, and to be uniform with the rest of the code-base